ts/src/digifinex.ts

```diff
@@ -2908,12 +2909,23 @@ export default class digifinex extends Exchange {
 
```

Comment:
@yzh-pelle can you please add a static request test validating the fix?

Reply:
Yeah, I'll do it